### PR TITLE
Adding nfdiv to nonprod

### PR DESCRIPTION
**Pull request by damongreen123**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: environments/demo/backend_lb_config.yaml [hunk 1]
```diff
@@ -191,6 +191,16 @@ gateways:
...
 
+      # FACT
+      - product: fact
+        component: api
+
+      # NFDIV
+      - product: nfdiv
+        component: nfdiv-orchestration-service
+      - product: nfdiv
+        component: nfdiv-orchestration-service
```
Comment (linusnorton) [inline environments/demo/backend_lb_config.yaml:202]: maintenance?

File: environments/ithc/backend_lb_config.yaml [hunk 2]
```diff
@@ -165,8 +165,14 @@ gateways:
...
       - product: fact
         component: api
-     
-    # xui
+
+    # NFDIV
+      - product: nfdiv
+        component: nfdiv-orchestration-service
+      - product: nfdiv
+        component: nfdiv-orchestration-service
```
Comment (linusnorton) [inline environments/ithc/backend_lb_config.yaml:173]: maintenance?

File: environments/stg/backend_lb_config.yaml [hunk 3]
```diff
@@ -214,14 +214,20 @@ gateways:
...
-    # FACT
+      # FACT
       - product: fact
         component: api
 
+      # NFDIV
+      - product: nfdiv
+        component: nfdiv-orchestration-service
+      - product: nfdiv
+        component: nfdiv-orchestration-service
```
Comment (linusnorton) [inline environments/stg/backend_lb_config.yaml:229]: maintenance?

File: environments/test/backend_lb_config.yaml [hunk 4]
```diff
@@ -190,6 +190,12 @@ gateways:
       - product: fact
         component: api
 
+    # NFDIV
+      - product: nfdiv
+        component: nfdiv-orchestration-service
+      - product: nfdiv
+        component: nfdiv-orchestration-service
```
Comment (linusnorton) [inline environments/test/backend_lb_config.yaml:197]: maintenance?

**Review by linusnorton — APPROVED** [review]

**Review by naikajah — APPROVED** [review]

**Pull request merged**